models/h2o_dai_regression_auto_insurance/app_h2o_mojo_pipeline_gunicorn.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright (c) 2020. Cognitive Scale Inc. All rights reserved.
+Licensed under CognitiveScale Example Code License https://github.com/CognitiveScale/cortex-certifai-examples/blob/master/LICENSE.md
+"""
+
+from certifai.model.sdk import SimpleModelWrapper
+
+
+class GermanCredit(SimpleModelWrapper):
+
+    @staticmethod
+    def __get_columns():
+        """
+        user-defined helper method. all user-defined methods must be prefixed by __
+        `__get_columns` is user defined helper method to list column names in order to re-create the dataframe for prediction
+        **Note**: all user defined methods must be declared inside the class as `instance methods`
+                  no user defined methods are allowed outside the scope of the class instantiating the `SimpleModelWrapper`
+                  for e.g. all methods must be declared inside the `GermanCredit` class (in this case) and referenced using `self`
+
+        :return: list of column names
+        """
+        columns = ['State Code',
+                   'Claim Amount',
+                   'Coverage',
+                   'Education',
+                   'EmploymentStatus',
+                   'Gender',
+                   'Income',
+                   'Location Code',
+                   'Marital Status',
+                   'Monthly Premium Auto',
+                   'Months Since Last Claim',
+                   'Months Since Policy Inception',
+                   'Number of Open Complaints',
+                   'Number of Policies',
+                   'Policy',
+                   'Claim Reason',
+                   'Sales Channel',
+                   'Vehicle Class',
+                   'Vehicle Size']
+        return columns
+
+    def set_global_imports(self):
+        """
+        overridden method to make external global imports
+        When using external imports override the method to provide necessary imports. Make sure to mark them `global` to
+        be used by certifai interpreter correctly. set once,use throughout.
+        :return: None
+        """
+        global dt
+        import datatable as dt
+
+    def _get_prediction_class(self, preds):
```

Comment:
I think we should rename the function to be _get_prediction given what it does in the regression case is return a probability not the class